services/bit/bit-components.service.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const Joi = require('joi')
+const { BaseJsonService } = require('..')
+const { metric } = require('../text-formatters')
+
+const ownerSchema = Joi.object({
+  payload: Joi.object({
+    totalComponents: Joi.number(),
+  }),
+}).required()
+
+module.exports = class bitComponents extends BaseJsonService {
+  async fetch({ owner, scope }) {
+    const url = `https://api.bit.dev/scope/${owner}/${scope}/`
+    return this._requestJson({
+      url,
+      schema: ownerSchema,
+      errorMessages: {
+        404: 'not found',
```

Comment:
`not found` is the default message on 404 errors, so you don't need to include this unless you want to customize that message (for example if you'd prefer 404 errors display something like `component not found`)